cmd/main.go

```diff
@@ -1,18 +1,40 @@
-// main.go
-
 package main
 
 import (
+	"api/config"
+	"api/database"
 	"api/delivery"
 	"api/repository"
 	"api/usecase"
+	"fmt"
 
 	"github.com/gin-gonic/gin"
 )
 
 func main() {
-	// Initialize the repository
-	bookRepository := repository.NewInMemoryBookRepository()
+	// Load configuration
+	config, err := config.LoadConfig()
+	if err != nil {
+		fmt.Println("Failed to load config:", err)
+		return
+	}
+
+	// Construct the connection string
+	connectionString := fmt.Sprintf("user=%s password=%s dbname=%s sslmode=%s host=%s port=%s",
+		config.DBUser, config.DBPassword, config.DBName, config.SSLMode, config.DBHost, config.DBPort)
+
+	// Connect to PostgreSQL
+	db, err := database.ConnectDB(connectionString)
```

Comment:
you can passing just `config` to `connectDB` and construct the connection string in database connection file

Reply:
done kang @iqbal167